src/setup/extract-configuration.go

```diff
@@ -89,17 +96,29 @@ func ExtractConfiguration(configFilePath string) Configuration {
 	if parsedConfig.Provider == "" {
 		parsedConfig.Provider = defaultProvider
 	}
+	if parsedConfig.Runtime == "" {
+		parsedConfig.Runtime = defaultRuntime
+	}
 
 	for index := range parsedConfig.SubExperiments {
 		if parsedConfig.SubExperiments[index].Function == "" {
 			parsedConfig.SubExperiments[index].Function = defaultFunction
 		}
+		if parsedConfig.SubExperiments[index].Handler == "" {
+			parsedConfig.SubExperiments[index].Handler = defaultHandler
+		}
+		if parsedConfig.SubExperiments[index].Runtime == "" {
```

Comment:
Can you change the subexperiment runtime default to parsedConfig.Runtime. So that we do not overwrite the experimet's default setting? (e.g. if  Configuration.Runtime is set to Python, we do not want to overwrite the subexperimet's runtime to Go.)

Reply:
Sure. Done at [43e56cb](https://github.com/vhive-serverless/STeLLAR/pull/268/commits/43e56cb6e2933c799c2ff34f22819b03bdf6fd55)